.gitignore

```diff
@@ -263,3 +263,9 @@ Dependencies/Assembly-CSharp-firstpass.dll
 Dependencies/Assembly-CSharp.dll
 /Dependencies/Assembly-CSharp-firstpass_publicized.dll
 /Dependencies/Assembly-CSharp_publicized.dll
+/DependenciesBZ/Assembly-CSharp_publicized.dll
+/DependenciesBZ/Assembly-CSharp-firstpass_publicized.dll
+/DependenciesBZ/Assembly-CSharp.dll
+/DependenciesBZ/Assembly-CSharp_publicized.dll
+/DependenciesBZ/Assembly-CSharp-firstpass.dll
+/DependenciesBZ/Assembly-CSharp-firstpass_publicized.dll
```

Comment:
Common dependencies should be stored in `Dependencies`
Subnautica dependencies should be stored in `Dependencies/Subnautica`
Below Zero dependencies should be stored in `Dependencies/BelowZero`